README.md

```diff
@@ -6,968 +6,16 @@
-  + [Easy to adopt](#easy-to-adopt)
-  + [Clean up an existing codebase](#clean-up-an-existing-codebase)
-  + [Ride the hype train](#ride-the-hype-train)
-* [How does it compare to ESLint (or TSLint, stylelint...)?](#how-does-it-compare-to-eslint-or-tslint-stylelint)
-* [Usage](#usage)
-  + [CLI](#cli)
-  + [ESLint](#eslint)
-  + [Pre-commit Hook](#pre-commit-hook)
-  + [API](#api)
-  + [Excluding code from formatting](#excluding-code-from-formatting)
-* [Options](#options)
-  + [Print Width](#print-width)
-  + [Tab Width](#tab-width)
-  + [Tabs](#tabs)
-  + [Semicolons](#semicolons)
-  + [Quotes](#quotes)
-  + [Trailing Commas](#trailing-commas)
-  + [Bracket Spacing](#bracket-spacing)
-  + [JSX Brackets](#jsx-brackets)
-  + [Range](#range)
-  + [Parser](#parser)
-  + [Filepath](#filepath)
-  + [Require pragma](#require-pragma)
-* [Configuration File](#configuration-file)
-  + [Basic Configuration](#basic-configuration)
-  + [Configuration Overrides](#configuration-overrides)
-  + [Configuration Schema](#configuration-schema)
-* [Editor Integration](#editor-integration)
-  + [Atom](#atom)
-  + [Emacs](#emacs)
-  + [Vim](#vim)
-  + [Visual Studio Code](#visual-studio-code)
-  + [Visual Studio](#visual-studio)
-  + [Sublime Text](#sublime-text)
-  + [JetBrains WebStorm, PHPStorm, PyCharm...](#jetbrains-webstorm-phpstorm-pycharm)
-* [Watching For Changes On The Command Line](#watching-for-changes-on-the-command-line)
-* [Language Support](#language-support)
-* [Related Projects](#related-projects)
-  + [ESLint Integrations](#eslint-integrations)
-  + [TSLint Integrations](#tslint-integrations)
-  + [stylelint Integrations](#stylelint-integrations)
-  + [Forks](#forks)
-  + [Misc](#misc)
-* [Technical Details](#technical-details)
-* [Badge](#badge)
-* [Contributing](#contributing)
-
-<!-- tocstop -->
-
-</details>
-
---------------------------------------------------------------------------------
-
-## What does Prettier do?
-
-Prettier takes your code and reprints it from scratch by taking the line length into account.
-
-For example, take the following code:
-
-```js
-foo(arg1, arg2, arg3, arg4);
-```
-
-It fits in a single line so it's going to stay as is. However, we've all run into this situation:
-
-```js
-foo(reallyLongArg(), omgSoManyParameters(), IShouldRefactorThis(), isThereSeriouslyAnotherOne());
-```
-
-Suddenly our previous format for calling function breaks down because this is too long. Prettier is going to do the painstaking work of reprinting it like that for you:
-
-```js
-foo(
-  reallyLongArg(),
-  omgSoManyParameters(),
-  IShouldRefactorThis(),
-  isThereSeriouslyAnotherOne()
-);
-```
-
-Prettier enforces a consistent code **style** (i.e. code formatting that won't affect the AST) across your entire codebase because it disregards the original styling[\*](#styling-footnote) by parsing it away and re-printing the parsed AST with its own rules that take the maximum line length
-into account, wrapping code when necessary.
-
```

Comment:
I'd keep this part here (not the title "What does prettier do?")

```
Prettier is an opinionated code formatter. It enforces a consistent style by parsing
your code and re-printing the parsed AST with its own rules that take the
maximum line length into account, wrapping code when necessary.

For example, take the following code:
<!-- and so on -->

You can try it on our interactive REPL: http://prettier.io/playground
```

Reply:
I’d still include the example to make it more clear what prettier does.

Reply:
Trying to avoid too much copy-pasting. We're doing this to remove the need to sync the docs.

However, I think it would be a great idea to write a _different_, more concise summary of Prettier in the README, with a basic example.

Reply:
Basically like this https://github.com/babel/babel#intro

Reply:
Yeah I'm just saying an example would be very good here, I don't really care which one :-)

I suggested keeping the one there just for the sake of simplicity